src/SprykerSdk/Acp/Schema/SchemaExtender.php

```diff
@@ -0,0 +1,155 @@
+     */
+    public function __construct(SchemaConverterInterface $schemaConverter, SchemaWriterInterface $schemaWriter)
+    {
+        $this->schemaConverter = $schemaConverter;
+        $this->schemaWriter = $schemaWriter;
+    }
+
+    /**
+     * @param \Transfer\CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+     *
+     * @return \Transfer\CreateDefaultEndpointsResponseTransfer
+     */
+    public function extendProjectSchema(CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer): CreateDefaultEndpointsResponseTransfer
+    {
+        $createDefaultEndpointsResponseTransfer = $this->addDefaultEndpointsToSchema($createDefaultEndpointsRequestTransfer);
+
+        $configurationSchemaGeneratedMessage = $this->convertConfigurationToSchema($createDefaultEndpointsRequestTransfer);
+
+        if ($configurationSchemaGeneratedMessage) {
+            $createDefaultEndpointsResponseTransfer->addError($configurationSchemaGeneratedMessage);
+        }
+
+        return $createDefaultEndpointsResponseTransfer;
+    }
+
+    /**
+     * @param \Transfer\CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+     *
+     * @return \Transfer\CreateDefaultEndpointsResponseTransfer
+     */
+    protected function addDefaultEndpointsToSchema(
+        CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+    ): CreateDefaultEndpointsResponseTransfer {
+        $result = new CreateDefaultEndpointsResponseTransfer();
+
+        $schema = $this->getDefaultEndpointsSchema($createDefaultEndpointsRequestTransfer);
+
+        $process = new Process([
+            'vendor/bin/syncapi',
+            'schema:openapi:update',
+            '-f',
+            $createDefaultEndpointsRequestTransfer->getSchemaFile(),
+            $this->schemaWriter->writeSchemaToJsonString($schema),
+        ]);
+        $process->run();
+
+        if (!$process->isSuccessful()) {
+            $result->addError($this->createMessage(sprintf('Updating openapi schema failed, error output: %s', $process->getErrorOutput())));
+
+            return $result;
+        }
+
+        if ($createDefaultEndpointsRequestTransfer->getAddLocal()) {
+            $registryFileCopied = $this->copyRegistryLocalFile($createDefaultEndpointsRequestTransfer);
+            if (!$registryFileCopied) {
+                $result->addError($this->createMessage('Copying local registry file failed'));
+
+                return $result;
+            }
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param \Transfer\CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer
+     *
+     * @return \cebe\openapi\spec\OpenApi
+     */
+    protected function getDefaultEndpointsSchema(CreateDefaultEndpointsRequestTransfer $createDefaultEndpointsRequestTransfer): OpenApi
+    {
+        $filePath = getcwd() . '/config/app/api/openapi/registry_reference_remote.yml';
```

Comment:
The current working directory won't work. Let's have the configuration tell us where the files are located (or at least where the `config` directory is placed).